styles/prosilver/template/manage/config_settings.html

```diff
@@ -0,0 +1,63 @@
+{% include 'manage/manage_header.html' %}
+
+<form id="config_settings" method="post" action="{{ U_ACTION }}" name="config_settings">
+	<div class="panel">
+		<div class="inner">
+			<fieldset>
+				{% for config, options in loops.options %}
+					<dl>
+						<dt><label for="{{ options.NAME }}">{{ lang(options.NAME|upper) }}:</label><br /><span>{{ lang(options.NAME|upper ~ '_EXPLAIN') }}</span></dt>
+						<dd>
+							{% if options.TYPE === 'string' %}
+								<input type="text" name="{{ options.NAME }}" id="{{ options.NAME }}" size="50" value="{{ options.VALUE }}" class="inputbox autowidth">
+							{% elseif options.TYPE === 'int' %}
+								<input type="number" name="{{ options.NAME }}" id="{{ options.NAME }}" min="0" max="9999999999" value="{{ options.VALUE }}" class="inputbox autowidth">
+							{% elseif options.TYPE === 'bool' %}
+								<input type="radio" name="{{ options.NAME }}" value="1" {% if options.VALUE %}checked="checked"{% endif %} /> {{ lang('YES') }}
+								<input type="radio" name="{{ options.NAME }}" value="0" {% if not options.VALUE %}checked="checked"{% endif %} /> {{ lang('NO') }}
+							{% elseif options.TYPE === 'array|string' %}
+								{% for subKey, subValue in options.VALUE %}
+									<label for="{{ options.NAME }}_{{ subKey }}"><input type="text" name="{{ options.NAME }}_{{ subKey }}" id="{{ options.NAME }}_{{ subKey }}" size="50" value="{{ subValue }}" class="inputbox autowidth"> {{ lang('TI_KEY_' ~ subKey) }}</label><br />
+								{% endfor %}
+							{% elseif options.TYPE === 'array|int' %}
+								{% for subKey, subValue in options.VALUE %}
+									<label for="{{ options.NAME }}_{{ subKey }}"><input type="number" name="{{ options.NAME }}_{{ subKey }}" id="{{ options.NAME }}_{{ subKey }}" min="0" max="9999999999" value="{{ subValue }}" class="inputbox autowidth"> {{ lang('TI_KEY_' ~ subKey) }} </label><br />
```

Comment:
Translations are set to `-1` so having `min="0"` here causes the validation to fail and can't save the form. Needs to be adjusted to `-1` or removed.

Reply:
Updated